base_classes/NXdisk_chopper.nxdl.xml

```diff
@@ -42,37 +57,65 @@
 		</enumeration>
 	</field>
 	<field name="rotation_speed" type="NX_FLOAT" units="NX_FREQUENCY">
-		<doc>chopper rotation speed</doc>
+		<doc>
+			Chopper rotation speed. Positive for anticlockwise rotation when
+			facing the sample, negative otherwise.
```

Comment:
I would make this "facing away from the source". Removes confusion if you ever put a chopper after the sample.

Reply:
:+1: done